contrib/lang/clojure/packages.el

```diff
@@ -77,7 +77,7 @@ which require an initialization must be listed explicitly in the list.")
         "mgv" 'cider-jump-to-var
         "mgr" 'cider-jump-to-resource
         "mge" 'cider-jump-to-compilation-error
-        "mgs" 'cider-jump
```

Comment:
Something that comes to mind...is there a convention that `mgs` is the canonical "goto source" chord for lisps? If so, we should keep `mgs` but just set it to use `cider-jump-var` instead.

Ping @syl20bnr